air-quality-backend/src/air_quality/api/main.py

```diff
@@ -1,8 +1,37 @@
-from fastapi import FastAPI
+from datetime import datetime
+from typing import Optional
+
+from fastapi import FastAPI, HTTPException
+
+from src.air_quality.database.forecasts import get_forecast_data_from_database
```

Comment:
Our imports shouldn't have src in them.

Reply:
It does seem to need it in multiple places, as it breaks if it doesn't.

Reply:
It shouldn't do that, where does it break? If you have a look through the rest of the project there are no src imports right now in main